lib/index.js

```diff
@@ -161,9 +161,11 @@ Segment.prototype.normalize = function(msg) {
   var global = exports.global;
   var query = global.location.search;
   var ctx = msg.context = msg.context || msg.options || {};
+  var amp = ctx.amp = ctx.amp || {};
```

Comment:
we should probably omit this too if we have no amp id? just so there's no `context.amp`